### PR TITLE
Sale details

### PR DESCRIPTION
**Pull request by Walterieux**
I created the interface for objects details, SaleDetails should be called by the activity that contains a list of all objects.

### CODE REVIEW

File: app/src/main/java/ch/epfl/polybazaar/SaleDetails.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    protected void onCreate(Bundle savedInstanceState) throws NullPointerException{
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_sale_details);
+
+        Bundle b = getIntent().getExtras();
+        if(b == null)
+            throw new NullPointerException();
+
+        //Set the image
+        //ImageView image = (ImageView)findViewById(R.id.imageView2);
```
Comment (JanMaxime) [inline app/src/main/java/ch/epfl/polybazaar/SaleDetails.java]: Remove unnecessary comments

File: app/src/main/java/ch/epfl/polybazaar/SaleDetails.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        //Set the price
+        TextView price_txt = (TextView)findViewById(R.id.price);
+        price_txt.setTextSize(20);
+        price_txt.setText(b.getString("price", "No Price"));
+
+
+        //Button ret = (Button) findViewById(R.id.ret_but);
+        //Button get_seller = (Button) findViewById(R.id.contact_sel);
+
+        //Return button action
```
Comment (JanMaxime) [inline app/src/main/java/ch/epfl/polybazaar/SaleDetails.java]: Remove unnecessary comments

File: app/src/main/java/ch/epfl/polybazaar/SaleDetails.java (hunk 1 above)
Comment (Walterieux) [reply app/src/main/java/ch/epfl/polybazaar/SaleDetails.java]: Done

File: app/src/main/java/ch/epfl/polybazaar/SaleDetails.java (hunk 2 above)
Comment (Walterieux) [reply app/src/main/java/ch/epfl/polybazaar/SaleDetails.java]: Done

**Pull request merged**